hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -449,6 +449,9 @@ public void refreshTimeline() throws IOException {
       case BULK_INSERT:
         writeStatusRDD = writeClient.bulkInsert(records, instantTime);
         break;
+      case INSERT_OVERWRITE:
```

Comment:
Should we also handle `INSERT_OVERWRITE_TABLE` case?

Reply:
added insert_overwrite_table case .
Also added `hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/dag/nodes/InsertOverwriteTableNode.java` for test

Reply:
Thanks! Could you also add a unit test for the two operation types in `TestHoodieDeltaStreamer`?

Reply:
@codope ,
I tried to add unit test cases, They are not passing. can you please help me with debugging what is missed?

Reply:
Thank you @codope for your help in debugging and pointing out the missing parts in `DeltaSync` :)